### ISSUE TITLE
ImportError: cannot import name 'print_formatted_text'

### ISSUE THREAD

**Issue by matthew-mcateer** [larq/zookeeper#140]
### Describe the bug

Larq Zoo cannot be imported. The stack trace points to a file in Zookeeper with the following error:

```python
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-2-0fabe46a7c79> in <module>()
      3 
      4 import larq
----> 5 import larq_zoo as lq

6 frames
/usr/local/lib/python3.6/dist-packages/larq_zoo/__init__.py in <module>()
----> 1 import larq_zoo.literature as literature
      2 import larq_zoo.sota as sota
      3 from larq_zoo.core.utils import decode_predictions
      4 from larq_zoo.training import datasets
      5 from larq_zoo.training.data import preprocess_input

/usr/local/lib/python3.6/dist-packages/larq_zoo/literature/__init__.py in <module>()
----> 1 from larq_zoo.literature.binary_alex_net import BinaryAlexNet
      2 from larq_zoo.literature.birealnet import BiRealNet
      3 from larq_zoo.literature.densenet import (
      4     BinaryDenseNet28,
      5     BinaryDenseNet37,

/usr/local/lib/python3.6/dist-packages/larq_zoo/literature/binary_alex_net.py in <module>()
      3 import larq as lq
      4 import tensorflow as tf
----> 5 from zookeeper import Field, factory
      6 
      7 from larq_zoo.core import utils

/usr/local/lib/python3.6/dist-packages/zookeeper/__init__.py in <module>()
----> 1 from zookeeper.core import (
      2     ComponentField,
      3     Field,
      4     PartialComponent,
      5     cli,

/usr/local/lib/python3.6/dist-packages/zookeeper/core/__init__.py in <module>()
----> 1 from zookeeper.core.cli import cli
      2 from zookeeper.core.component import component, configure
      3 from zookeeper.core.factory import factory
      4 from zookeeper.core.field import ComponentField, Field
      5 from zookeeper.core.partial_component import PartialComponent

/usr/local/lib/python3.6/dist-packages/zookeeper/core/cli.py in <module>()
      3 import click
      4 
----> 5 from zookeeper.core.utils import convert_to_snake_case, parse_value_from_string
      6 
      7 

/usr/local/lib/python3.6/dist-packages/zookeeper/core/utils.py in <module>()
      6 import click
      7 import typeguard
----> 8 from prompt_toolkit import print_formatted_text, prompt
      9 
     10 

ImportError: cannot import name 'print_formatted_text'

---------------------------------------------------------------------------
NOTE: If your import is failing due to a missing package, you can
manually install dependencies using either !pip or !apt.

To view examples of installing some common dependencies, click the
"Open Examples" button below.
---------------------------------------------------------------------------
```

### To Reproduce

```python
# Installing Larq and Larq-zoo in a Colab notebook
!pip -q install larq
!pip install larq-zoo

import larq
import larq_zoo as lq
```

### Expected behavior

`larq_zoo` being imported successfully with no other output from the cell in Google Colab

### Environment

TensorFlow version: 2.2.0-rc3
Zookeeper version: 1.0.0
Larq zoo: 1.0.1
Larq: 0.9.4

**Comment by AdamHillier** [comment]
Hey, thank you very much for filing this issue. I have confirmed that this happens in a fresh colab notebook.

I got this additional bit of logging when installing the two packages:
```
ERROR: jupyter-console 5.2.0 has requirement prompt-toolkit<2.0.0,>=1.0.0, but you'll have prompt-toolkit 3.0.5 which is incompatible.
ERROR: ipython 5.5.0 has requirement prompt-toolkit<2.0.0,>=1.0.4, but you'll have prompt-toolkit 3.0.5 which is incompatible.
Installing collected packages: prompt-toolkit, zookeeper, larq-zoo
  Found existing installation: prompt-toolkit 1.0.18
    Uninstalling prompt-toolkit-1.0.18:
      Successfully uninstalled prompt-toolkit-1.0.18
Successfully installed larq-zoo-1.0.1 prompt-toolkit-3.0.5 zookeeper-1.0.0

WARNING: Upgrading ipython, ipykernel, tornado, prompt-toolkit or pyzmq can
cause your runtime to repeatedly crash or behave in unexpected ways and is not
recommended. If your runtime won't connect or execute code, you can reset it
with "Factory reset runtime" from the "Runtime" menu.
WARNING: prompt-toolkit > 1.0.16 is incompatible with IPython < 7.0
WARNING: The following packages were previously imported in this runtime:
  [prompt_toolkit]
Run "pip install -U ipython" before restarting to avoid repeated crashes.
```

Zookeeper has a dependency on [`prompt>=2.0.10`](https://github.com/larq/zookeeper/blob/7dd56d34bd226b320f232a33e8d0d81761bc933b/setup.py#L23), which seems to conflict with the versions of `ipython` and `jupyter-console` installed in colab by default.

I will investigate what we can do about this.

**Comment by AdamHillier** [comment]
In #141 we should hopefully remove `prompt-toolkit` as a Zookeeper dependency, which is actually something I'd been planning to do for a while.

**Comment by AdamHillier** [comment]
We just released v1.0.1 of Zookeeper with a fix in it, and I've just confirmed that this has resolved the issue on colab.
